docs/add-new-integration.md

```diff
@@ -0,0 +1,89 @@
+            description: number of idle threads in the pool
+        tag:
+          - name: jetty.threadpool.type
+            value: ${type}
+          - name: jetty.instance.id
+            value: ${id}
+    ```
+6. Once you have all the YAML files are the new integration ready, you need to enable monitoring in the application to be monitored.
+This depends on the data source type.
+    * For JMX, in case of standalone monitoring, make sure the appropriate JMX startup arguments are passed to 
+      the Java process to be monitored.
+    * For DB, make sure proper permissions are set up for the monitoring user for the table to be queried.
+    * For JSON, make sure the URL is accessible and proper permissions are set up.
+      
+    For Jetty, add the below arguments to Java startup process to enable standalone agent to query JMX metrics:
+    
+    ```bash
+    -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=3000 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false
+    ``` 
+    Setup new monitor in standalone using below `setup-spm` command:
+    ```bash
+    sudo bash /opt/spm/bin/setup-spm  --app-token <app-token>  --app-type jetty  --agent-type standalone
+    ```
+    The above command will start a standalone monitor and will request for JMX metrics from the Jetty server running on localhost in port 3000.
+    <app-token> - App Token should point to Sematext App Token if you are sending metrics to Sematext. 
+    For other Influx endpoints you can specify a hexadecimal value with format xxxxxxxx--xxxx-xxxx-xxxx-xxxxxxxxxxxx 
+    e.g. d0add28a-0a0f-46b2-9e1e-4928db5200e7.
+    
+    You can also pass custom arguments from `setup-spm` command and use them in YAML definitions. For more info refer to
+    [How to pass custom arguments to my integration](metrics-yaml-format.md#specifying-variables-in-yaml)
+7. By default, the agent sends the collected metrics to Sematext. You can configure a different Influx compatible destination
+   by changing the following properties in `/opt/spm/properties/agent.properties` file:
+   * `server_base_url` - Base URL of the destination server. e.g. `http://192.168.0.4:8086`
+   * `metrics_endpoint` - Path to send the metrics. This will be appended with `server_base_url` to form the complete URL.
+       Default value is `/write?db=metrics`. You can update this property to send metrics to different endpoint 
+       or to specify username/password for InfluxDB. e.g. `/write?db=mydb&u=user&p=pass`
+8. For each monitor setup, the agent generates 2 log files under `/opt/spm/spm-monitor/logs/applications/<app-token>/default`
+    directory.
+    * `spm-monitor-*.log` - Contains the logs generated by the agent during monitoring. You can check this logs for any errors during monitoring like invalid YAML, cannot fetch metrics, etc.
+    * `spm-monitor-stats-*.log` - Contains the Influx compatible metrics that are sent to the receiver. 
```

Comment:
I use `destination` to refer to the server. I will change`receiver` to `destination`.  Also, will change the description since the file contain actual output sent (not just influx encoded)